Tasks/Common/webdeployment-common/deployusingmsdeploy.ts

```diff
@@ -47,17 +47,18 @@ export async function DeployUsingMSDeploy(webDeployPkg, webAppName, publishingPr
     
     try {
         var shouldContinue = true;
```

Comment:
We should remove this variable as not being in use now.

Reply:
removed.